playingaway.R

```diff
@@ -8,134 +8,157 @@ library(ggplot2)
 library(ggthemes)
 library(tidyr)
 
+# Get team data ----
+team_state_csv <-
+    "team,state
```

Comment:
Possibly. Rename `state` here to `state_abbrev` and then merge with  this:

```
states <- read_csv("state_abbrev,state
    NSW,New South Wales
    TAS,Tasmania,
    WA,Western Australia
    NT,Northern Territory
    SA,South Australia
    QLD,Queensland
    ACT,Australian Capital Territory
    VIC,Other (New Zealand, etc.)") 
```

I'd be surprised if you need to use `state` from the underlying source very often: teams and venues have states, but what else? (Games are played by teams at venues, so games don't have states directly.)